### ISSUE TITLE
Sorting by size and time is wrong.

### ISSUE THREAD

**Issue by EV-soft** [prasathmani/tinyfilemanager#197]
Version 2.3.8:
Column size looks cluttered with left adjustment, and is wrong, when numbers are converted by units.
Column time is wrong, when year is located in the middle of the text string.

**Comment by EV-soft** [comment]
One solution to fix sorting, is add a hidden prefix to TD-output:

Line 1764 - Change:
`
$modif = date(FM_DATETIME_FORMAT, filemtime($path . '/' . $f));`
To:

```
$ftime = filemtime($path . '/' . $f);
$modif = date(FM_DATETIME_FORMAT, $ftime);
```
Line 1786- Change:
```
                    <td><?php if ($calc_folder) { echo fm_get_directorysize($path . '/' . $f); } else { echo "Folder"; } ?></td>
                    <td><?php echo $modif ?></td>
```
To:
```
                    <td><?php if ($calc_folder) { $foldsize = fm_get_directorysize($path . '/' . $f); echo '<span style="display:none;">'.str_pad($foldsize, 18, "0", STR_PAD_LEFT).' </span>'.fm_get_filesize($foldsize); } else { echo "Folder"; } ?></td>
                    <td><?php echo '<span style="display:none;">'.$ftime.'</span>'.$modif ?></td>
```
Line 1809- Change:
`                $modif = date(FM_DATETIME_FORMAT, filemtime($path . '/' . $f));`

To:
```
               $ftime = filemtime($path . '/' . $f); //  Sort criteria
                $modif = date(FM_DATETIME_FORMAT, $ftime);
```

Line 1836- Change:
`                        <?php echo $filesize; ?>`
To:
`                        <?php echo '<span style="display:none;">'.str_pad($filesize_raw, 18, "0", STR_PAD_LEFT).'</span>'.$filesize; ?>`

Line 1838- Change:
`                    <td><?php echo $modif ?></td>`
To:
`                    <td><?php echo '<span style="display:none;">'.$ftime.'</span>'.$modif ?></td>`

**Another defect**
For Table: Zebra-background on rows to display:
Line 3142- add:
`        .table-bordered tr:nth-child(even) { background: #f8f8f8; } `

**Comment by EV-soft** [comment]
I have worked on other improvements, as well as those already mentioned.
View source documents in [ZIP file](https://www.sky-box.dk/tinyFM/TFM-2.3.8-EV.zip)
Watch the demo here: [DEMO](https://www.sky-box.dk/tinyFM/TFM-v2.3.8.php?p=)
Comments have been added to all corrections. (Search: Translate / EV-soft / more... )
Of particular note is the routine for the maintenance of language translations. (See Settings)
Complete Danish language is included i translation.json
